src/hash_table.py

```diff
@@ -0,0 +1,25 @@
+# HASH TABLE (HT)
+# 
+# CodeFellows 401d5
+# Submission Date:
+#
+# Authors:  Colin Lamont <https://github.com/chamberi>
+#           Ben Shields <https://github.com/iamrobinhood12345>
+#
+# URL:
+
+
+"""
+The Mailroom module allows the user to
+track donations and format emails for donors.
+The user can write tailored thank you emails
+given donor names and donation amounts.
+The user can create a report that is
+a list of donor names and their corresponding donation
+histories, arranged by in order of the total amount
+donated.
+The user can add a donor name to this list.
+The user can quit the program from Main Menu, and
+may return to Main Menu at any time.
```

Comment:
Not sure that the hash table tracks donations and writes tailored thank you emails...